src/splitInteger.test.js

```diff
@@ -4,18 +4,20 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+    expect(splitInteger(8, 1)).toEqual([8]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+    expect(splitInteger(6, 2)).toEqual([3, 3]);
```

Comment:
The description for this test case does not match the expectation. The description suggests splitting into 1 part, but the test is actually splitting 6 into 2 parts. Update the description to accurately reflect the test case.